src/components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+import { ReactNode } from "react";
+import { ChevronRightIcon, HomeIcon } from '@heroicons/react/20/solid';
+import { Link } from "@yext/pages/components";
+import classNames from "classnames";
+
+interface Breadcrumb {
```

Comment:
Choose interface or type and be consistent throughout. Directly below this you use a type.